### ISSUE TITLE
SCC pods terminate immediately after deploy of WLO app

### ISSUE THREAD

**Issue by tpickett1** [WASdev/websphere-liberty-operator#376]
When deploying a new app and you have N number of SCCs configured, there will be N SCC pods with start but terminate immediately.  Possible there are other resources not yet ready(timing issue).  Then, once those N SCC pods terminate, there are N new SCC pods started and start fine.  

Waiting on those pods to terminate does delay the overall deployment and pod startup.  I have not seen where the delay time is more than a few secs, but it would be nice to have that time back for startup.

I was able to reproduce this using the sample app.
